e2e-tests/README.md

```diff
@@ -202,18 +221,36 @@ yarn run desktop
 
 Using Terminal 1:
 
+Destroy the previously launched enos scenario:
+
 ```bash
-# Community Edition
+# aws / ce
 enos scenario destroy e2e_ui_aws builder:local
+```
 
-# Enterprise Edition
+```bash
+# aws / enterprise
 enos scenario destroy e2e_ui_aws_ent builder:local
 ```
 
+```bash
+# docker / ce
+enos scenario destroy e2e_ui_docker builder:local
+```
+
+```bash
+# docker / enterprise
+enos scenario destroy e2e_ui_docker_ent builder:local
+```
+
 After all the steps pass, you should see a `Enos operations finished!`.
 
 ## Developing Tests
 
+### Test names and tagging
```

Comment:
FYI: I plan on updating the tags to use the new syntax outlined here (https://dev.to/playwright/tagging-your-playwright-tests-3omm). Not a huge change, but just wanted to share.